src/sdk/clients/createBicoPaymasterClient.test.ts

```diff
@@ -133,4 +141,144 @@ describe.runIf(paymasterTruthy())("bico.paymaster", async () => {
     // No gas fees were paid, so the balance should have decreased only by 1n
     expect(finalBalance).toBe(initialBalance - 1n)
   })
+
+  test("should use token paymaster to pay for gas fees, use max approval", async () => {
+    const nexusClient = await createNexusClient({
+      signer: account,
+      chain,
+      paymaster: createBicoPaymasterClient({
+        transport: http(paymasterUrl)
+      }),
+      paymasterContext: {
+        mode: "ERC20",
+        tokenInfo: {
+          feeTokenAddress: baseSepoliaUSDCAddress
+        }
+      },
+      transport: http(),
+      bundlerTransport: http(bundlerUrl),
+      ...testParams
+    })
+
+    const initialBalance = await publicClient.getBalance({
+      address: nexusAccountAddress
+    })
+
+    const hash = await nexusClient.sendTransaction({
+      to: recipientAddress,
+      value: 1n,
+      chain: baseSepolia
```

Comment:
There are other places in tests where the chain is not required during sendTransaction though right? Can you investigate to make sure that this isn't because of your changes?

Reply:
No it's not related to my changes, we only use "sendTransaction({calls: [tx]})" inside our SDK, we haven't used "sendTransaction(tx)" anywhere, default viem behaviour is to require chain as well.